.github/workflows/test.yml

```diff
@@ -17,7 +17,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - platform: ubuntu-latest
+            os: ubuntu-latest
+          - platform: windows-latest
+            os: windows-latest
+          - platform: macos-latest
+            os: [self-hosted, macos, arm64]
```

Comment:
Just out of curiosity, do you know if it would be possible to just specify `os: [ubuntu-latest, windows-latest, [self-hosted, macos, arm64]]`?

Reply:
Turns out [it is possible](https://github.com/phylum-dev/cli/actions/runs/3229981438)